beacon/utils/db.py

```diff
@@ -223,7 +222,7 @@ async def fetch_variants(connection,
                          individual_stable_id=None):
     LOG.info('Retrieving viral variant information')
 
-    dollars = ", ".join([ f"${i}" for i in range(1, 21)]) # 1..20
+    dollars = ", ".join([ f"${i}" for i in range(1, 22)]) # 1..21
```

Comment:
If we give it a bit of time, I'm sure we can reach the postgres limit!
What is it? 65536 ?

Reply:
https://stackoverflow.com/a/16586401